.github/workflows/checks.yml

```diff
@@ -0,0 +1,51 @@
+name: Run checks
+
+on: [push]
+
+permissions:
+  contents: read
+
+env:
+  NOTIFY_ENVIRONMENT: test
+  FLASK_APP: application.py
+  FLASK_ENV: development
+  WERKZEUG_DEBUG_PIN: off
+  REDIS_URL: redis://adminredis:6379/0
+  DEV_REDIS_URL: redis://adminredis:6379/0
+  REDIS_ENABLED: False
+  ANTIVIRUS_ENABLED: 0
+  NODE_VERSION: 16.15.1
+  ADMIN_CLIENT_ID: notify-admin
+  ADMIN_CLIENT_USERNAME: notify-admin
+  ADMIN_CLIENT_SECRET: dev-notify-secret-key
+  GOVUK_ALERTS_CLIENT_ID: govuk-alerts
+  ADMIN_BASE_URL: http://localhost:6012
+  API_HOST_NAME: http://localhost:6011
+  DEV_API_HOST_NAME: http://localhost:6011
+  AWS_REGION: us-west-2
+  BASIC_AUTH_USERNAME: curiousabout
+  BASIC_AUTH_PASSWORD: the10xnotifybeta
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Install container dependencies
+      run: |
+        sudo apt-get update \
+        && sudo apt-get install -y --no-install-recommends \
+        libcurl4-openssl-dev
+    - uses: actions/checkout@v3
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v3
+      with:
+        python-version: "3.9"
+    - name: Install application dependencies
+      run: make bootstrap
+    # - name: Run style checks
+    #   run: flake8 .
+    # - name: Check imports alphabetized
+    #   run: isort --check-only ./app ./tests
+    - name: Run tests
+      run: pytest -n4 --maxfail=10
```

Comment:
[suggestion] I think we should be running `npm test` in CI as well

Reply:
[committed](https://github.com/18F/notifications-admin/pull/20/commits/cdd7b42f21f63335c4ca3a104be935e701e19721)